.github/workflows/universalnrp-test.yml

```diff
@@ -7,9 +7,27 @@ on:
     - cron: '0 20 * * *' # Every day at 12pm PST (UTC-8)
 
 jobs:
+  package:
+    name: Package
+    uses: ./.github/workflows/package-build.yml
+    strategy:
+      matrix:
+        target:
+          [
+            { os: ubuntu, version: 14.04, arch: amd64, dist: trusty, package-type: DEB },
+          ]
+    with:
+      target: ${{ matrix.target.os }}-${{ matrix.target.version }}
+      arch: ${{ matrix.target.arch }}
+      artifact: nrp-test-package
```

Comment:
>artifact: nrp-test-package [](http://example.com/codeflow?start=6&length=26)

Please do not do this. Instead, build correctly named SSH and ASB packages that will be both released as artifacts and tested here, under those exact names. 

The package names matter and the MOF inside them has that name set in.

Reply:
Discussed offline: The artifact name still maintains the original policy package names internally. 

\+ restored the `nrp-build.yml` workflow